distributed/cli/dask_ssh.py

```diff
@@ -212,15 +214,21 @@ def main(
     for i, host in enumerate(hostnames):  # pragma: no cover
         print(f"  {i}: {host}")
     print(f"\nscheduler node: {scheduler}:{scheduler_port}")  # pragma: no cover
-    print("---------------------------------------------------------------\n\n")  # pragma: no cover
+    print(
+        "---------------------------------------------------------------\n\n"
+    )  # pragma: no cover
 
     # Monitor the output of remote processes.  This blocks until the user issues a KeyboardInterrupt.
     c.monitor_remote_processes()
```

Comment:
why doesn't line 222` need a `# pragma: no cover`? - for line 222 to be covered then 219 needs to have been covered

Reply:
Please remove the pragmas. We shouldn't spend too much time on this